fix/shooting.lua

```diff
@@ -0,0 +1,34 @@
+-- Fixes shooting/throwing options stacking on each other, causing null pointers and crashes
+--@ enable = true
```

Comment:
this script does not implement the enabled API, so this marker won't do anything.
relevant docs here: https://docs.dfhack.org/en/latest/docs/dev/Lua%20API.html#enabling-and-disabling-scripts